install.php

```diff
@@ -12,7 +12,7 @@
 	exit;
 }
 
-require_once __DIR__ . "/vendor/autoload.php";
+require_once "/vendor/autoload.php";
```

Comment:
P.S. What I was affraid of: http://stackoverflow.com/questions/25438287/how-to-configure-a-composer-package-to-be-globally-installed#answer-25438654

Will have to handle dependency case manually in Elgg files.